chart/templates/olm/clusterserviceversion.yaml

```diff
@@ -3,6 +3,12 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  labels:
```

Comment:
We should push this change upstream to ecosystem with correct values and here in the fork, we should just update ossm/values.yaml

Reply:
https://github.com/istio-ecosystem/sail-operator/pull/650

Reply:
so I can close this PR right?  it will be propagated by automator bot